packages/edit-site/src/components/block-editor/back-button.js

```diff
@@ -24,11 +24,14 @@ function BackButton() {
 	const isTemplatePart = location.params.postType === TEMPLATE_PART_POST_TYPE;
 	const isNavigationMenu = location.params.postType === NAVIGATION_POST_TYPE;
 	const isPattern = location.params.postType === PATTERN_TYPES.user;
-	const previousTemplateId = location.state?.fromTemplateId;
 
-	const isFocusMode = isTemplatePart || isNavigationMenu || isPattern;
+	const isFocusMode =
+		location.params.focusMode ||
```

Comment:
It feels like we can probably remove this focusMode from the url no? Why did we keep it?

Reply:
Oh so it looks like a new argument in the URL. I don't understand why we had to introduce it, can you explain?

Reply:
If I understand properly, this argument was added to avoid having to keep a local history of focused entities. I think it can work for now. (But please confirm)

Reply:
The comment thread related to this [is here](https://github.com/WordPress/gutenberg/pull/57700#discussion_r1462687050) - but yes, this was introduced as a way to avoid the complexities of tracking the history of focused entities in combination with the existing routing in the site editor (which makes it more complicated than the simple implementation in the post editor). As noted in that comment I may be overlooking a simpler solution, but given that `focusMode` has been in the code base for at least 6 months and achieves what we want, using this seems like a lower risk approach for 6.5 than tracking the history, and doesn't cause any backwards compat issues for following up with router changes post 6.5.